pandas/_testing.py

```diff
@@ -1703,6 +1710,7 @@ def assert_frame_equal(
                 obj=f'{obj}.iloc[:, {i}] (column name="{col}")',
                 rtol=rtol,
                 atol=atol,
+                check_index=False,
             )
```

Comment:
we have a lot of kwargs already.  would it be viable to call assert_(ea|numpy)_array_equal on lcol._values and rcol._values?

Reply:
Probably.
But I looked inside ``assert_series_equal`` and noticed that there are a lot of cases covered, different dtypes, etc.
So, I thought that it would be safer to parametrize.
I will try to find another way, as you suggest.

Reply:
@jbrockmendel, I am afraid I can only suggest the following approach for now.

Split ``assert_series_equal`` into ``assert_index_equal`` and ``_assert_series_values_equal``.
The latter one is the same as the original ``assert_series_equal``, but with the ``assert_index_equal`` removed from there.

Then we can use ``_assert_series_values_equal`` in the loop inside ``assert_frame_equal``.

Reply:
works for me

Reply:
hmm i actually liked your prior implementation (`check_index`) but i guess this is fine.